app/src/main/java/com/housemate/classes/User.java

```diff
@@ -103,6 +103,29 @@ public void login() throws RuntimeException {
         }
     }
 
+    public void deleteUser() throws RuntimeException{
```

Comment:
In a later commit, we should make use of the initiateTransaction and mapToJson methods Adam recently introduced in HTTPSDataSender here to simplify the code.